### DISCUSSION TITLE
Documentation/Code Inconsistency

### DISCUSSION THREAD

**Discussion by makeclean** [idaholab/moose#19147] [Q&A General] [answered]
The documentation for [Output] in the Exodus section - https://mooseframework.inl.gov/source/outputs/Exodus.html says that options for execute_on are 'NONE, INITIAL, LINEAR, NONLINEAR, TIMESTEP_END, TIMESTEP_BEGIN, FINAL, FAILED, CUSTOM, ALWAYS', however when running the code, ALL isn't allowed, the options are "NONE INITIAL LINEAR NONLINEAR TIMESTEP_END TIMESTEP_BEGIN FINAL FAILED CUSTOM".

**Comment by GiudGiud** [comment, marked as answer]
Hello

ALL isnt in the list, but ALWAYS is. it's accepted as a parameter. However I ve just tested it and it doesn't trigger on a simple example. 
I'll see if others pitch in on this. This is probably something to fix though

fyi best documentation on execute_on I could find is here:
https://mooseframework.inl.gov/bison/source/interfaces/SetupInterface.html

Guillaume

**Reply by GiudGiud** [reply]
Added an issue
https://github.com/idaholab/moose/issues/19255
so I dont forget about it
